thirdparty/gperftools-2.0/src/gperftools/tcmalloc.h

```diff
@@ -92,7 +92,7 @@ extern "C" {
 
   PERFTOOLS_DLL_DECL void tc_malloc_stats(void) __THROW;
   PERFTOOLS_DLL_DECL int tc_mallopt(int cmd, int value) __THROW;
-#if 1
+#if 0
```

Comment:
Why?

Reply:
Didn't intentionally change this, will revert.